max-wave.lua

```diff
@@ -0,0 +1,51 @@
+--Dynamically limit the next immigration wave
+--By Loci, modified by Fleeting Frames
+--[====[
+
+max-wave
+========
+Set the population cap to the lesser of current_pop + wave_size or max_pop.
+Use with the `repeat` command to set a rolling immigration limit.
+
```

Comment:
you should make it clear that this script *overrides* the maximum population set in `data/init/d_init.txt`